mscore/menus.cpp

```diff
@@ -2,7 +2,7 @@
 //  MuseScore
 //  Linux Music Score Editor
 //
-//  Copyright (C) 2002-2016 Werner Schweer and others
+//  Copyright (C) 2020 MuseScore BVBA and others
```

Comment:
As you're not changing anything else in this file, I don't think the copyright needs to get changed, not as part of this PR at least

Reply:
Oh, I'm sorry. Modified the wrong file by accident. Actually I got a bit lost in the code, but I've almost figured it out. I'll be sending the final PR soon, after fixing all the issues.